keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
+static bool shift_pressed;
+
+static int lower = (int) KC_A;
+static int upper = (int) KC_Z;
+
+uint16_t random_keycode(void) {
+  return lower + (rand() % (upper - lower + 1));
```

Comment:
It probably doesn't matter for this use case, but this way of generating random numbers in a range has very poor uniformity (it favors lower numbers most of the time). Here is a better way to do it:
```suggestion
  return rand()/(RAND_MAX+1.0) * (UPPER-LOWER+1) + LOWER;
```
(`UPPER` and `LOWER` in case you decide to change them to preprocessor constants like I suggested.)